Documentation/HelpInside/InternalNotes/Index.rst

```diff
@@ -0,0 +1,36 @@
+:navigation-title: Internal Notes
+
+..  include:: /Includes.rst.txt
+..  _internal-notes:
+
+===================================
+Internal notes in the TYPO3 backend
+===================================
+
+..  note::
+    These are only available if :composer:`typo3/cms-sys-note` is installed
+    and your administrator granted you the necessary user rights.
+
+Internal notes, also called system notes, are small text notes that are
+displayed in the `Page module <https://docs.typo3.org/permalink/t3editors:content-creating>`_
+and `List module <https://docs.typo3.org/permalink/t3editors:using-the-list-module-effectively>`_.
+
+You can use them to write notes to yourself or other editors. Administrators
+can use them to write notes to you as well.
+
+They look like this:
+
+..  figure:: /Images/ManualScreenshots/Page/PageWithSysNote.png
+    :alt: Screenshot of a TODO note on the top of the "Page" backend module
+
+    If you have sufficient permissions you can also edit or delete the note.
+
+
```

Comment:
The both images looks a bit glued together. I prefer a small text in between to give them more space to breathe. Further it looks like the image below belongs to the saying "They look like this" which is not the cause.

```suggestion
    The following image shows a detailed view of the TYPO3 backend toolbar, specifically in
    the context of the "Page" or "List" module. On the right side of the toolbar are a number
    of buttons, each providing different functionality. The button highlighted in this screenshot
    is which allows the user to create a new internal note.

```